db/migrate/20151220112512_add_devise_to_users.rb

```diff
@@ -16,8 +16,13 @@ def self.up
       t.integer  :sign_in_count, default: 0, null: false
       t.datetime :current_sign_in_at
       t.datetime :last_sign_in_at
-      t.inet     :current_sign_in_ip
-      t.inet     :last_sign_in_ip
+      if Rails.env.production?
```

Comment:
`inet` is postgres-specific, so this is a dirty hack to get around and deal with both sqlite3 and postgres.